content/shmem_ctx_get_team.tex

```diff
@@ -0,0 +1,45 @@
+\apisummary{
+  Retrieve the team associated with the communication context.
+}
+
+\begin{apidefinition}
+
+  \begin{Csynopsis}
+int @\FuncDecl{shmem\_ctx\_get\_team}@(shmem_ctx_t ctx, shmem_team_t *team);
+  \end{Csynopsis}
+
+  \begin{apiarguments}
+
+    \apiargument{IN}{ctx}{
+      A handle to a communication context.
+    }
+
+    \apiargument{OUT}{team}{
+      A pointer to a handle to the associated \ac{PE} team.
+    }
+
+  \end{apiarguments}
+
+  \apidescription{
+    The \FUNC{shmem\_ctx\_get\_team} routine returns a handle to the \ac{PE}
+    team associated with the specified communication context \VAR{ctx}.
+    The team handle is returned through the pointer argument \VAR{team}.
+
+    If \VAR{ctx} is the default context, the returned team is guaranteed
+    to be \CONST{SHMEM\_TEAM\_WORLD}.
+
+    If \VAR{ctx} is an invalid context, the argument \VAR{team} is not
+    modified and a value of \CONST{-1} is returned.
```

Comment:
Do we want to restrict the return value to -1 exactly? Or just specify a value less than 0, in case the implementation wants to specify some different error codes?

Reply:
I'm not super picky. I think that's a fair point to raise for discussion.

I can create a `FeedbackRequested` environment in LaTeX so we can merge PRs for tomorrow's call, but have the design questions documented in the working draft.

Reply:
Sounds good to me. I have got other functions documented with error return value less than 0 rather than -1, so I can add those to a FeedbackRequested environment as well.